README.md

```diff
@@ -70,6 +70,24 @@ gem install sendgrid-ruby
 - [Ruby-HTTP-Client](https://github.com/sendgrid/ruby-http-client)
 - [Sinatra](http://www.sinatrarb.com/) - this is only needed if you plan to process [Inbound Email](#inbound).
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
If you are doing this step, then all you need to do is open and edit the file by inserting your API KEY.

The next section `environment variable` is redundant.